### PR TITLE
Fix Calibrate App .env.sample and update README.md

### LINKED ISSUE #891: Fix Calibrate App .env.sample mistake and update respective README.md
https://github.com/airqo-platform/AirQo-frontend/blob/2cc2c90097a4d86257304942b197790dcb010a7a/calibrate/.env.sample#L6
- The value of `REACT_APP_AUTHORIZATION_TOKEN` is meant to be plainly a JWT token without the "JWT" part. Update the [.env.sample](https://github.com/airqo-platform/AirQo-frontend/blob/staging/calibrate/.env.sample) file from `REACT_APP_AUTHORIZATION_TOKEN=JWT ey123abc` to `REACT_APP_AUTHORIZATION_TOKEN=ey123abc`

- The `REACT_APP_AUTHORIZATION_TOKEN` is an [Authentication](https://staging-docs.airqo.net/#/api/?id=authentication) token used to access all API endpoints and so it is always needed. The other task of this issue is to update the [Calibrate App README](https://github.com/airqo-platform/AirQo-frontend/blob/staging/calibrate/README.md) to;
  - Let the users know of the existence of the .env.sample file, from which they can create their personal `.env` file when contributing to the app
  - Add the [documentation link](https://staging-docs.airqo.net/#/../api/users?id=login) for users on how to get an authentication token.

#### Additional context
- The .env.sample details should be included as prerequisite in the README

### PR DESCRIPTION
**Pull request by thomasjvu**
#### Summary of Changes (What does this PR do?)
- Closes #891 
- This lets the reader know of the existence of the .env.sample file and how to obtain their own authentication token.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [x] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* View .env.sample file for accuracy and check if the README.md file is satisfactory

#### What are the relevant tickets?
- #891 

#### Screenshots (optional)

### CODE REVIEW

File: calibrate/README.md [hunk 1]
```diff
@@ -7,6 +7,20 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
...
+
+- `.env` file
+```
+PORT=5000
+NODE_PATH=src/
+REACT_APP_BASE_URL=http://staging-platform.airqo.net/api/v1/
+REACT_APP_DOCS_BASE_URL=https://staging-docs.airqo.net/#/
+REACT_APP_BASE_CALIBRATE_URL=
+REACT_APP_AUTHORIZATION_TOKEN=ey123abc
+```
```
Comment (123MwanjeMike) [inline calibrate/README.md:21]: Could we remove this part since it's already included in the `.env.sample` file

**Review by 123MwanjeMike — CHANGES_REQUESTED** [review]
Great work @thomasjvu 
One small request tho.

File: calibrate/README.md (hunk 1 above)
Comment (thomasjvu) [reply calibrate/README.md]: Hi @123MwanjeMike.
I just added another commit that makes this change. Please check it out 😄

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**